build/BuildSteps.cs

```diff
@@ -121,11 +120,6 @@ public static void DotnetPublishForZips()
                 {
                     RemoveLanguageWorkers(outputPath);
```

Comment:
Do we have any use case where we need to remove language workers in this version/branch of core tools? If not, let's remove it.

Reply:
The minified version is used by VS and the OOP version used to run .NET Isolated shouldn't have any workers, so that is still a valid use case.